sky/cli.py

```diff
@@ -3463,23 +3463,29 @@ def _get_kubernetes_realtime_gpu_table(
             ])
         return realtime_gpu_table
 
-    # TODO(zhwu): this needs to run on remote server.
```

Comment:
I removed this TODO since I think the current behavior is okay since this function only does formatting, which varies in different clients, e.g. CLI and Web UI. Would be good to keep this at client-side. Wdyt? @Michaelvll 

cc @DanielZhangQD

Reply:
Also renamed the function to `_format_kubernetes_node_info` for clarity

Reply:
Ahh, this TODO was mainly to reduce the RTT by node_info_dict = sdk.stream_and_get(sdk.kubernetes_node_info(context=context))` below. It is fine to remove : )